docs/RFCs/0002-build-process.md

```diff
@@ -0,0 +1,86 @@
+# RFC 0002: Building process proposal
+
+## Context
+As described in the [RFC](https://github.com/DataDog/libdatadog/blob/main/docs/RFCs/0001-delivering-different-features.md)
+about delivering different features the current build process depends heavily on shell scripts which makes it too 
+difficult in terms of managning the process itself like handling errors, dependendencies and features. Also shell
+scripts by themselves poses some disadvantages like portability and maintanibilty.
+
+This solution aims to get rid of those problems so the whole procedure depends as much as possible of built-in features 
+of cargo making it less error prone and also improving its maintanibilty.
```

Comment:
Agree🤔 
I would say that shell script allows to have the same command everywhere instead of searching for the command in the yaml file.

Do not get me wrong, I'm not against just saying that they are useful (in some case).

Now, I see that we can build, generate binding in one-line command thanks for that🙇‍♂️

Reply:
> Now, I see that we can build, generate bindings
    ^-------------------------- am I correct ?

Reply:
Yep, in this case the cons were clearly outbalancing the pros.